src/components/templates/Audience/Audience.tsx

```diff
@@ -313,27 +314,22 @@ export const Audience: React.FunctionComponent = () => {
     };
 
     const onSubmit = async (data: ChatOutDataType) => {
+      if (!venueId || !userWithId) return;
+
       setIsShoutSent(true);
-      user &&
-        dispatch(
-          addReaction({
-            venueId,
-            reaction: createReaction(
-              { reaction: "messageToTheBand", text: data.text },
-              user
-            ),
-          })
-        );
+
+      dispatch(
+        addReaction({
+          venueId,
+          reaction: createTextReaction(data.text, userWithId),
+        })
+      );
```

Comment:
Should this be moved into the hook with dispatch wrapping in the hook? So that we can use a clean `addTextReaction` fn here & everywhere else?

Reply:
This is sort of OOS of this PR. IMO it belongs within the scope of @yurii-lubynets's https://github.com/sparkletown/internal-sparkle-issues/issues/637 / https://github.com/sparkletown/sparkle/pull/1339

Reply:
(referenced this in https://github.com/sparkletown/internal-sparkle-issues/issues/637#issuecomment-824456706)